cinema/serializers.py

```diff
@@ -22,3 +22,54 @@
         instance.save()
 
         return instance
+
+
+class ActorSerializer(serializers.Serializer):
```

Comment:
Consider using `serializers.ModelSerializer` for `ActorSerializer` to simplify the code and automatically handle model fields.